mods/cnc/rules/structures.yaml

```
@@ -234,6 +248,10 @@ PROC:
 		Footprint: _x_ xxx +++ ===
 		Dimensions: 3,4
 		LocalCenterOffset: 0,-512,0
+	Encyclopedia:
+		Description: The refinery stores and processes the alien material and resource, Tiberium. Harvesters will collect Tiberium in the field and deposit it at the refinery, where it is converted into credits.\n\n The refinery stores 1000 credits worth of Tiberium and immediately deploys a harvester once constructed. Can act as a drop off for 3 harvesters.
```

```suggestion
		Description: The refinery stores and processes the alien material and resource, Tiberium. Harvesters will collect Tiberium in the field and deposit it at the refinery, where it is converted into credits.\n\nThe refinery stores 1000 credits worth of Tiberium and immediately deploys a harvester once constructed. Can act as a drop off for 3 harvesters.
```

Afaik it can also act as drop off point for any number of harvesters. Harvesters will just look for another refinery if there are already 3 at this refinery.

---

Was more angling the time it takes to deposit, it can't support more than 3 close mining harvesters. You know, general game advice. Fixed.